src/Behat/Behat/HelperContainer/BuiltInServiceContainer.php

```diff
@@ -12,7 +12,7 @@
 
 use Behat\Behat\HelperContainer\Exception\ServiceNotFoundException;
 use Behat\Behat\HelperContainer\Exception\WrongServicesConfigurationException;
-use Interop\Container\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
Basically we need to require `container-interop` from `1.2`, which will extend the thing. Then we indeed need to check if the psr is implemented